resolvers/locationPin.js

```diff
@@ -27,9 +27,11 @@ const locationPinResolvers = {
   LocationPin: {
     eventId: (parent) => {
       validators.validateRole(Object.values(Roles), validators.demoRole);
-
       return db.event.findByPk(parent.eventId);
     },
+    ccpParentId: (parent) => {
+      return db.collectionPoint.findByPk(parent.ccpParentId);
+    },
```

Comment:
don't recall how this works exactly but would we need a null check here? `ccpParentId` is the first foreign key we've introduced that ISN'T mandatory, so just want to make sure we're doing it properly

Reply:
This section of the code is referring to the nested information that we display on GraphQL. We don't need to do a null check for CCP because we are allowing pins to have empty `ccpId` fields, as you can see in the GraphQL schema.